script.js

```diff
@@ -1,103 +1,130 @@
-// Get the modal
+const ghost = document.querySelector('.ghost');
+const pumpkin = document.querySelector('.pumpkin');
+const reaper = document.querySelector('.grim_reaper');
+const lineEight = document.querySelector('.line_eight');
+const start = document.querySelector('.start_area');
+const collision = document.querySelectorAll('.collision');
 const winModal = document.querySelector("#gameWinPopup");
 const loseModal = document.querySelector("#gameLosePopup");
+const winningDoor = document.querySelector('.winningSquare');
 
-//close the modal because there is no restart button
-document.querySelector('#closeLoseModal').addEventListener('click', () => {
-    loseModal.style.display = "none";
-})
+//animation section, shouldnt be in function as consts need to be globally accessible
+const ghostMove = ghost.animate([
+    {transform: 'translateY(0px)'},
+    {transform: 'translateY(150px)'},
+    {transform: 'translateY(0px)'}
+], {
+    duration: 3000,
+    iterations: Infinity
+});
+ghostMove.pause();
+
+const pumpkinMove = pumpkin.animate([
+    {transform: 'translateX(0px)'},
+    {transform: 'translateX(120px)'},
+    {transform: 'translateX(0px)'},
+    {transform: 'translateX(-140px)'},
+    {transform: 'translateX(0px)'}
+], {
+    duration: 6000,
+    iterations: Infinity
+});
+pumpkinMove.pause();
+
+const reaperMove = reaper.animate([
+    {transform: 'translateY(0px)'},
+    {transform: 'translateY(100px)'},
+    {transform: 'translateY(0px)'}
+], {
+    duration: 3000,
+    iterations: Infinity
+});
+reaperMove.pause();
 
-document.querySelector('#closeWinModal').addEventListener('click', () => {
+// code to make splash screen mouse hover
+function homeScreenGhost () {
```

Comment:
Both the comment and the function name don't really explain what this does - change to something like `addSplashScreenGhostAnimation()`

Reply:
Also in this case it's not really worth creating a function for this just to call it once immediately after - you should just run the code without having it in a function. Reasons why this is okay:
- The code runs only once
- This project is very front-endsy and we're not doing unit testing on it (unusual project)